### PR TITLE
Update README with new "How to play"-instructions

### LINKED ISSUE #201: README must be updated with the new way to play once argus-server "frontend-switch" is merged and released
See docs for Uninett/Argus#899

### PR DESCRIPTION
**Pull request by hmpf**
Closes #201.

### CODE REVIEW

**Review by elfjes — CHANGES_REQUESTED** [review]
maybe also add instructions on how to use `argus_htmx.appconfig.APP_SETTINGS` with `update_settings`? and refer to `argus.htmx.settings`

**Review by lunkwill42 — APPROVED** [review]
Nothing to add except that I agree with @elfjes' suggestion :)

File: README.rst [hunk 1]
```diff
@@ -33,118 +33,70 @@ Configure
...
-                    "social_django.context_processors.backends",
-                    "social_django.context_processors.login_redirect",
-                    "argus_htmx.context_processors.theme_via_session",
-                    "argus_htmx.context_processors.datetime_format_via_session",
-                ],
-            },
-        }
-    ]
+(You might want to switch out ``from argus.site.settings.backend import *`` with
+``from argus.site.settings.dev import *``.)
```
Comment (johannaengland) [inline README.rst:48]: Why? I would like an explanation here when it is needed/helpful.

File: README.rst [hunk 2]
```diff
@@ -33,118 +33,70 @@ Configure
...
-    PUBLIC_URLS = [
-        "/accounts/login/",
-        "/api/",
-    ]
+You make a JSON-file which is read into your settings via one of two
+environment variables.
 
-If you use a shell script to control ``manage.py``, add the following
-environment variable::
+In order to add apps and settings that *extends* ``argus-server`` and this
```
Comment (johannaengland) [inline README.rst:84]:
```suggestion
In order to add apps and settings that *extend* ``argus-server`` and this
```

File: README.rst [hunk 3]
```diff
@@ -33,118 +33,70 @@ Configure
...
-          "urlpatterns_module": "debug_toolbar.urls"
-        }
-      }
-    ]
+    export ARGUS_OVERRIDING_APPS=`cat overriding.json`
+
+Have a look at the contents of ``argus_htmx.appconfig._app_settings`` for an
+example of what you can set this way.
+
+You can merge your urlpatterns with the apps's urlpatterns via the
```
Comment (johannaengland) [inline README.rst:97]:
```suggestion
You can merge your urlpatterns with the apps' urlpatterns via the
```

**Review by johannaengland — CHANGES_REQUESTED** [review]

**Review by johannaengland — APPROVED** [review]

**Pull request merged**